tools/devops/automation/templates/sign-and-notarized/funnel.yml

```diff
@@ -46,10 +46,12 @@ jobs:
   steps:
 
   # DO NOT USE THE checkout.yml template. The reason is that the template changes the hash which results in a problem with the artifacts scripts
-  - checkout: self          # https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema#checkout
-    clean: true             # Executes: git clean -ffdx && git reset --hard HEAD
-    submodules: recursive
-    path: s/xamarin-macios
+  #
```

Comment:
```suggestion
```
Removing the empty comment - I'm guessing the previous commit message doesn't apply here anymore?